tests/SCF/test_SCF.py

```diff
@@ -1,19 +1,94 @@
 import pytest
 import SCF
+import numpy as np
+from pyscf import gto  # PySCF is a quantum chemistry python module
 
 
-def test_calc_nuclear_repulsion_energy(mol_h2o):
-    assert True
-#    assert SCF.calc_nuclear_repulsion_energy(mol_h2o) == 8.00236706181077,\
-#        "Nuclear Repulsion Energy Test (H2O) Failed"
+def test_all_functions():
+    # Input Data
+    mol_h2o = gto.M(unit="Bohr",

```

Comment:
The fixture already does this and you should just pass in the fixture to each of the test functions.